detailed_guides/INSTALLATION.md

```diff
@@ -80,54 +80,74 @@ To connect with ssh, run the following command in terminal:
   
 It will then prompt you for your password, which you should enter, and then you will be logged in.   
   
-After login, we need to install the various dependencies that the bot needs. To do this, we will run the following commands:  
+# After login, we need to install the various dependencies that the bot needs. To do this, we will run the following commands:  
 ```shell
-git clone https://github.com/Kav-K/GPT3Discord.git
-cd GPT3Discord/  
-# Install system packages (python)  
-sudo apt-get update
-sudo apt install software-properties-common
-sudo add-apt-repository ppa:deadsnakes/ppa
-sudo apt install python3.9
-sudo apt install python3.9-distutils # If this doesn't work, try sudo apt install python3-distutils  
+git clone https://github.com/Kav-K/GPT3Discord.git```
+
+```cd GPT3Discord/```
+
+# Install system packages (python) 
+```sudo apt-get updat```
```

Comment:
Can you fix the typo here? There are also some formatting issues:
![image](https://user-images.githubusercontent.com/21161101/222601167-6e20367f-a47b-4f42-85af-f72822cb3dca.png)